app/routers/analytics.py

```
@@ -0,0 +1,36 @@
+from ..schemas import ModerationSummary
+
+router = APIRouter(prefix="/api/v1", tags=["analytics"])
+
+
+async def get_session():
+    async with async_session() as session:
+        yield session
+
+
+@router.get("/analytics/summary", response_model=ModerationSummary)
+async def analytics_summary(user: str, session: AsyncSession = Depends(get_session)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)